modules/ROOT/pages/authentication-authorization/manage-users.adoc

```diff
@@ -743,33 +797,67 @@ The `SET PASSWORD` part is only optional if it directly follows the `SET PASSWOR
 * `SET HOME DATABASE` can be used to configure a home database for a user.
 A home database will be resolved if it is either pointing to a database or a database alias.
 If no home database is set, the DBMS default database is used as the home database for the user.
+[role=label--new-5.24]
+* One or more `SET AUTH` clauses can be used to set xref:authentication-authorization/auth-objects.adoc[Auth Objects], which define authentication / authorization providers for the user. This might be used to configure external auth providers like LDAP or OIDC, but can also be used as an alternative way to set the native (password-based) auth settings like `SET PASSWORD` and `SET PASSWORD CHANGE REQUIRED`. TODO: Insert links to the LDAP and OIDC examples.
 * `REMOVE HOME DATABASE` is used to unset the home database for a user.
 This results in the DBMS default database being used as the home database for the user.
+[role=label--new-5.24]
+* `REMOVE AUTH` is used to remove one or many xref:authentication-authorization/auth-objects.adoc[Auth Object(s)] from a user. It cannot be used such that a user would have no Auth Objects.
+* `REMOVE ALL AUTH` is used to remove all existing xref:authentication-authorization/auth-objects.adoc[Auth Objects] from a user and replace them with new ones. It must be used in conjunction with at least one `SET AUTH` clause.
```

Comment:
>from a user and replace them with new ones

make it sound like the remove clause adds new ones, might be better to just say removes them and that it is invalid without at least one SET AUTH to add new ones as replacements

Reply:
done, thanks